pkgs/development/python-modules/jupyterlab_server/default.nix

```diff
@@ -52,6 +54,9 @@ buildPythonPackage rec {
     rm -r tests/translations/
   '';
 
+  # https://github.com/jupyterlab/jupyterlab_server/blob/v2.15.2/pyproject.toml#L61
+  doCheck = openapi-core.version == "0.14.2";
```

Comment:
This will likely not be updated and bit rot and also we usually only have on version in nixpkgs. Can we do that different?

Reply:
We would have to put `doCheck = false;` until upstream fixes the tests then.